platforms/hyperledger-fabric/charts/peernode/values.yaml

```diff
@@ -11,13 +11,13 @@ metadata:
   images:
     #Provide the valid image name and version for fabric couchdb
     #Eg. couchdb: hyperledger/fabric-couchdb:0.4.14
-    couchdb: 
+    couchdb: hyperledger/fabric-couchdb:0.4.14
     #Provide the valid image name and version for fabric peer
     #Eg. hyperledger/fabric-peer:1.4.0
-    peer: 
+    peer: hyperledger/fabric-peer:1.4.0
     #Provide the valid image name and version to read certificates from vault server 
     #Eg. alpineutils: hyperledgerlabs/alpine-utils:1.0
-    alpineutils: 
+    alpineutils: hyperledgerlabs/alpine-utils:1.0
```

Comment:
confirm what the code uses, should be ghcr.io/hyperledger, same for other charts using alpine

Reply:
Fixed it across all the Helm charts.